content/blog/2022/12/camel320-whatsnew/index.md

```diff
@@ -0,0 +1,122 @@
+
+---
+title: "Apache Camel 3.20 What's New"
+date: 2022-12-21
+authors: [davsclaus]
+categories: ["Releases"]
+preview: Details of what we have done in the Camel 3.20 release.
+---
+
+Apache Camel 3.20 has just been [released](/blog/2022/12/RELEASE-3.20.0/).
+
+This release introduces a set of new features and noticeable improvements that we will cover in this blog post.
+
+## Camel Core
+
+The Split EIP has been optimized to perform faster and reduced overhead, when splitting by a single separator char.
```

Comment:
```suggestion
The Split EIP has been optimized to perform faster and reduced overhead when splitting by a literal or a pattern.
```

Reply:
it actually covers more than a single char if you are referring to https://github.com/apache/camel/pull/8570